src/classes/GiftBatchAggregateGroupName.cls

```diff
@@ -0,0 +1,5 @@
+public enum GiftBatchAggregateGroupName {
```

Comment:
Even though this file is tiny, should probably still have the copyright at the top.